### ISSUE TITLE
Naming for Noto Sans TC (Chinese Traditional)

### ISSUE THREAD

**Issue by hfhchan** [google/fonts#274]
The Noto Sans TC (Chinese Traditional) font uses glyphs that are not suitable for the Hong Kong locale, despite having the same coverage.  The font design neither follows commercially acceptable glyph standards in Hong Kong nor follows the Hong Kong government's "Guidelines on Character Glyphs for Chinese Computer Systems".

FYI, the glyphs for HB0, HB1 and HB2 currently in the Unicode Standard are NOT supplied by Hong Kong but instead are substituted as placeholders.

Please rename the font Noto Sans TC (Chinese Traditional) to Noto Sans TW (Chinese Traditional).

The same issue has been raised over in the Source Han Sans repository (https://github.com/adobe-fonts/source-han-sans/issues/18) and the font has been renamed to TW since 1.001.

Please adjust the Noto font accordingly.

**Comment by hfhchan** [comment]
The standard orthography currently used in Taiwan is the result of extensive language engineering and differs drastically to the orthography generally used in Greater China area for hundreds of years.

The Kangxi dictionary, which is used as the base of ISO10646 CJK standardization, regarded as the "authoritative" orthography in the Qing dynasty, is by far more faithfully represented in modern Korean or Japanese orthography. 

The modern Taiwanese orthography places ridiculously strict requirements on the orthography utilizing forms that were rarely used or even invented but reflect to resemble the etymology better as of known knowledge at that time. Taiwan may proclaim their use of Chinese as traditional due to the choice of variants, but the orthography is definitely not.

**Comment by hfhchan** [comment]
That said, the Source Han Sans fonts now supports both the Taiwanese or Hong Kong orthography as different locales.

If possible, it would greatly aid language coverage for Hong Kong if the changes from Source Han Sans were ported back to Noto Sans CJK.

If there are currently no plans to update Noto Sans, since the original fonts for Noto Sans TC was designed according to the Taiwanese orthography, which is neither traditional, nor suitable for Hong Kong the other majority region using Traditional Chinese, the font name should be adjusted to reflect so.

**Comment by chiyi4488** [comment]
I agree with you, but I don't think it should be modified to Noto Sans TW. Because TW stands for Taiwan, but Taiwan is not traditional Chinese (literally). If you really want to be clear, it would be better to use Noto Sans TC (TW) and Noto Sans TC (HK).

It is undeniable that it is indeed traditional Chinese.
